Utilities/ScanSubscription.ps1

```diff
@@ -0,0 +1,390 @@
+# Description: This script is used for checking enabled VM sizes and VM core quota in the given azure subscription.
+# It will upload subscription id and unavailable VM/insufficient VM core to db
+# This script will be executed by 'Penguinator-Validate-Azure-Subscription-Info' pipeline
+# This script should be under .\lisav2\Utilities directory
+
+Param
+(
+    [String] $clientSecret,
+    [String] $clientID,
+    [String] $tenantID,
+    [String] $subID,
+    [String] $lisaTestCasesXMLPath = '.\lisav2\XML\TestCases',
+    [String] $lisaVMConfigsPath = '.\lisav2\XML\VMConfigurations',
+    [String] $penguinatorSecureFilePath
+)
```

Comment:
can we add one more parameter for parallel count, the required quota may change per parallel count.
Say the parallel count is n, then the minimal quota for a size is the sum of n cases with top n required cores.

Reply:
Sure. 
To clarify this:
1. what should be the default n? 2?
2. after getting the sum of top n required cores, do we still want to round up to closest 100s?

Reply:
currently please set n default to 4.
Yes we still need to round up to 10